libraries/chain/block_header_state.cpp

```diff
@@ -369,18 +369,17 @@ block_header_state block_header_state::next(const signed_block_header& h, valida
    // retrieve protocol_feature_activation from incoming block header extension
    // -------------------------------------------------------------------------
    vector<digest_type> new_protocol_feature_activations;
-   if( exts.count(protocol_feature_activation::extension_id() > 0) ) {
-      auto  pfa_entry = exts.lower_bound(protocol_feature_activation::extension_id());
+   if (auto  pfa_entry = exts.find(protocol_feature_activation::extension_id()); pfa_entry != exts.end()) {
```

Comment:
The behavior of `find()` is different then `lower_bound()` when the mutlimap contains identical keys. Fortunately it seems all these extensions currently `enforce_unique = true` so there appears to be no way to concoct multiple extensions of a single id in the header. (if we did allow that, I think this would introduce a determinism problem here)